modules/getting-started-cli-connecting-a-database.adoc

```diff
@@ -6,7 +6,8 @@
 [id="getting-started-cli-connecting-database_{context}"]
 = Deploying the database application
 
-The following procedure deploys `mongodb-nationalparks`, which is a MongoDB database that will hold the national park location information.
+[role="_abstract"]
+Deploy the `mongodb-nationalparks` MongoDB database application, which will hold the national park location information.
```

Comment:
I think maybe try to get this a bit more focused on the general learning goal, then add the scenario specifics
```suggestion
Deploy a MongoDB database application to contain the information that your application requires. For this tutorial, you will deploy a database application called `mongodb-nationalparks` that holds national park location information.
```

Reply:
Updated!